src/CrossCompiler.cmake

```
@@ -356,7 +356,7 @@ macro(enable_cross_compiler)
     else()
       if(NOT DEFINED EMSCRIPTEN_ROOT)
         include(FetchContent)
-        message(STATUS "fetch emscripten repo. ...")
+        message(WARNING "fetch emscripten repo main branch. ...")
```

I am not sure why this is a warning
```suggestion
        message(STATUS "fetch emscripten repo main branch. ...")
```

---

I want to show that is a moving target!